producer/contentProducer_test.go

```diff
@@ -0,0 +1,141 @@
+package producer
+
+import (
+	"encoding/json"
+	"errors"
+	"github.com/Financial-Times/message-queue-go-producer/producer"
+	tidutils "github.com/Financial-Times/transactionid-utils-go"
+	"github.com/Financial-Times/uuid-utils-go"
+	"github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
+	_ "github.com/stretchr/testify/assert"
```

Comment:
You can remove the import solely for side-effects as you also imported the whole package above.